scalding-beam/src/test/scala/com/twitter/scalding/beam_backend/BeamBackendTests.scala

```diff
@@ -132,11 +122,25 @@ class BeamBackendTests extends FunSuite with BeforeAndAfter {
     assert(test.isFailure)
   }
 
+  test("SumByLocalKeys"){
+    beamMatchesSeq(
+      TypedPipe
+        .from(0 to 5)
+        .map(x => (x, x))
+        .flatMapValues(x => 0 to x)
+        .sumByLocalKeys(new Semigroup[Int] {
+          override def plus(x: Int, y: Int): Int = x + y
+        }),
+      Seq((0, 0), (1, 1), (2, 3), (3, 6), (4, 10), (5, 15)),
+      Config.empty.setMapSideAggregationThreshold(5)
+    )
+  }
+
   private def getContents(path: String, prefix: String): List[String] = {
     new File(path).listFiles.flatMap(file => {
-      if (file.getPath.startsWith(prefix)) {
+      if(file.getPath.startsWith(prefix)){
         Source.fromFile(file).getLines().flatMap(line => line.split("\\s+").toList)
-      } else List.empty[String]
+      }else List.empty[String]
```

Comment:
why the unusual spacing?

Reply:
I use two different things for formatting the code, the first one is the reformat tool from IntelliJ, but that doesn't really work to format the code (sometimes add spaces), this one just optimize the imports, so before update the PR's I'm running scalariformFormat command (this one doesn't optimize imports) in sbt, and this one always format the code but always in a different way, reordering functions or removing/adding spaces etc... That's the reason for all weird unnecessary changes between updates

Reply:
we should use scalafmt.... but that's another issue.